tests/test_user_fusion.py

```diff
@@ -0,0 +1,84 @@
+import pytest_asyncio
+from fastapi.testclient import TestClient
+
+from app.core import models_core
+from app.core.groups.groups_type import GroupType
+from tests.commons import (
+    create_api_access_token,
+    create_user_with_groups,
+)
+
+admin_user: models_core.CoreUser
+student_user_to_delete: models_core.CoreUser
+student_user_to_keep: models_core.CoreUser
+
+token_admin_user: str
+token_student_user: str
+
+
+FABRISTPP_EMAIL_1 = "fabristpp.eclair1@etu.ec-lyon.fr"
+FABRISTPP_EMAIL_2 = "fabristpp.eclair3@ecl21.ec-lyon.fr"
+
+
+@pytest_asyncio.fixture(scope="module", autouse=True)
+async def init_objects() -> None:
```

Comment:
Could you add a few objects involving relationships? It would be great to have:
– an object with a relationship that is unique but not a primary key
– an object with a relationship that is neither unique nor a primary key